stable/appmesh-manager/README.md

```diff
@@ -4,9 +4,9 @@ App Mesh manager Helm chart for Kubernetes
 
 ## Prerequisites
 
-Note: App Mesh manager is a release candidate. Please use it for testing purpose only as it has backward incompatible changes with v0.5.0. Upgrade instructions will be provided before the final release.
+**Note** App Mesh manager is a release candidate. Please use it for testing purpose only as it has backward incompatible changes with v0.5.0. Upgrade instructions will be provided before the final release.
 
-Note: If you wish to use App Mesh preview features, you can add `-preview` to the published imge. e.g. `v1.0.0-rc4-preview`
+**Note** If you wish to use App Mesh preview features, you can add `-preview` to the published imge. e.g. `v1.0.0-rc4-preview`
```

Comment:
imge -> image

Reply:
Thanks. Was a left from the last I touched this. Fixed!